### ISSUE TITLE
Update missing image descriptions identified during front end audit

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4217]
Non-text content should have a text alternative.  The following images did not have appropriate alt-text when reviewed during #4194.  They need to be updated.

- [ ] [Sample 1](https://ijwwe.uwp.co.uk/article/id/9711/), item 24. "Janeway logo final" is read as the title of the logo at hte bottom. The 'final' is unhelpful. <title>Janeway-Logo-Final</title>
- [ ] [Sample 3](https://politicalphilosophyjournal.org/article/id/15481/), item 12. image description is article title which not only doesn't describe the actual image, but the title then follows as a heading - consider a 'describedby' reference to the heading instead.
- [ ] [Sample 3](https://politicalphilosophyjournal.org/article/id/15481/), item 51. OLH logo does not have alt-text
- [ ] [Sample 4](https://www.iastatedigitalpress.com/mmb/article/id/16903/), item 20. figure 1 - voiceover attempts to read the text within the png. but as the order is about the visual layout, this makes no sense.
- [ ] [Sample 4](https://www.iastatedigitalpress.com/mmb/article/id/16903/), item 42. press logo at bottom of the page is missing an alt-text so the image file title is read out - this is a collection of numbers and letters. This is due to a missing closing " following the class attribute, which prevents the alt="" attribute being recognised <img src="/media/cover_images/d78e50a8-3542-4429-b049-20261739ee56.svg" class=" alt=" "="">.
- [ ] [Sample 5.1](https://www.iastatedigitalpress.com/mmb/), item 3. should cover image be set to decorative? Is "[Journal name] cover image" as a description helpful in this context/position on the page?
- [ ] [Sample 5.2](https://www.bstjournal.com/), item 5. latest news image is not described.

**Comment by mauromsl** [comment]
closed by https://github.com/openlibhums/janeway/pull/5027
